browser/main/NoteList/index.js

```diff
@@ -324,6 +330,23 @@ class NoteList extends React.Component {
     }
   }
 
+  jumpToTop() {
+    if (this.notes == null || this.notes.length === 0) {
```

Comment:
Yes. But I don't know what the difference between `===` and `_.isNull` thus I used `===` because I knew it.